pkgs/servers/forked-daapd/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv,
+  fetchFromGitHub,
+  avahi,
+  antlr3,
+  autoconf,
+  autoreconfHook,
+  curl,
+  ffmpeg,
+  gawk,
+  gettext,
+  gnutls,
+  gperf,
+  json_c,
+  minixml,
+  openssl,
+  libantlr3c,
+  libconfuse,
+  libevent,
+  libgcrypt,
+  libgpgerror,
+  libplist,
+  libsodium,
+  libunistring,
+  libuv,
+  libwebsockets,
+  pkg-config,
+  protobufc,
+  pulseaudio,
+  sqlite,
+  zlib,
+  spotifySupport ? false, libspotify ? null
+}:
+
+let
+  version = "27.2";
+  name = "forked-daapd";
+in stdenv.mkDerivation {
+  inherit name;
```

Comment:
```suggestion
stdenv.mkDerivation {
  pname = "forked-daapd";
  version = "27.2";
```

Reply:
It also needs to be a recursive attrset:

```
stdenv.mkDerivation rec {
```

Reply:
Thanks. That works.